pkg/controller/servicebindingrequest/reconciler_test.go

```diff
@@ -205,3 +205,78 @@ func TestReconcilerReconcileUsingVolumes(t *testing.T) {
 		require.Equal(t, reconcilerName, volumes[0].VolumeSource.Secret.SecretName)
 	})
 }
+
+func TestReconcilerGenericBinding(t *testing.T) {
+	ctx := context.TODO()
+	backingServiceResourceRef := "test-using-volumes"
```

Comment:
We are not using volumes for this test case so we should rename "test-using-volumes"

Reply:
fixed 78118e8